cmake/modules/FindCPP_TASKFLOW.cmake

```diff
@@ -0,0 +1,58 @@
+## ---------------------------------------------------------------------
+##
+## Copyright (C) 2020 by the deal.II authors
+##
+## This file is part of the deal.II library.
+##
+## The deal.II library is free software; you can use it, redistribute
+## it, and/or modify it under the terms of the GNU Lesser General
+## Public License as published by the Free Software Foundation; either
+## version 2.1 of the License, or (at your option) any later version.
+## The full text of the license can be found in the file LICENSE.md at
+## the top level directory of deal.II.
+##
+## ---------------------------------------------------------------------
+
+#
+# Try to find the Threading Building Blocks library
+#
+# This module exports
+#
+#   CPP_TASKFLOW_INCLUDE_DIRS
```

Comment:
include_dir or include_dirs?

Reply:
Yes, `DIRS` (because this it the one you should use).